src/exceptionHandling.js

```diff
@@ -0,0 +1,21 @@
+import { Console } from "@woowacourse/mission-utils";
+import { ERROR_MESSAGES } from "./constant.js";
+
+export const inputCarsException = (cars) => {
+  if (cars.length === 1 && cars[0] === "") {
+    throw new Error(ERROR_MESSAGES.NO_CAR_INPUT);
+  }
+
+  if (cars.some((carName) => carName.length > 5)) {
+    throw new Error(ERROR_MESSAGES.EXCEED_CAR_LENGTH);
+  }
+};
+
+export const inputCountException = (count) => {
+  if (isNaN(count)) {
+    throw new Error(ERROR_MESSAGES.INVALID_COUNT_INPUT);
+  }
+  if (count == null || count <= 0) {
+    throw new Error(ERROR_MESSAGES.NO_COUNT);
```

Comment:
```suggestion
  if (count <= 0) {
```
위에서 `isNaN()` 함수로 숫자인지를 검사했으니 `count <= 0` 조건만 있어도 될 것 같네요!

Reply:
꼼꼼한 리뷰 감사해요 덕분에 많은 것을 알아갑니다👍🏻😁🥹